Utils/py_utils.py

```diff
@@ -0,0 +1,5 @@
+import os
```

Comment:
Utils/__init is empty (the above file). Why is it pushed?

Reply:
this indicate to python that this is a python package

Reply:
oh oh yes. usually we add something like
```
"""
Utils module

This metafile includes all the functionality that will be exposed
when you install this module
"""
from ...

```

so lets have at least a doc string

Reply:
@mbenita-Cyberark what do you think?